### PR TITLE
gnomeExtensions.paperwm: init at 36.0

### PR DESCRIPTION
**Pull request by zowoq**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc @hedning @jtojnar @worldofpeace

### CODE REVIEW

File: pkgs/desktops/gnome-3/extensions/paperwm/default.nix
Comment (hedning) [inline pkgs/desktops/gnome-3/extensions/paperwm/default.nix]: The build phase doesn't do anything as upstream ships compiled schemas.
```suggestion

   dontBuild = true;
```

**Comment by hedning** [comment]
Note, I'll release paperwm-36.0 soon, which have gnome 3.36 support. But I guess it doesn't hurt to get this in.

File: pkgs/desktops/gnome-3/extensions/paperwm/default.nix
Comment (zowoq) [reply pkgs/desktops/gnome-3/extensions/paperwm/default.nix]: Updated.

**Comment by worldofpeace** [comment]
@hedning Probably should add yourself as a maintainer of this package :grin:

**Comment by hedning** [comment]
Yep, that's probably.

How do you push to PRs again (I now I've done it before, but seem to have forgotten)?

**Comment by zowoq** [comment]
@hedning I've added you as a maintainer.

**Comment by jtojnar** [comment]
`git push https://github.com/zowoq/nixpkgs.git HEAD:paperwm -f`

Also if you use `gitAndTool.gh` the you can do `gh pr checkout 81785` and just `git push -f` (it will set the remote for the branch correctly.

Might also need https://github.com/cli/cli/issues/546#issuecomment-591088766

**Comment by hedning** [comment]
OK, I just released [36.0](https://github.com/paperwm/PaperWM/releases/tag/36.0), lets start it there :)

**Comment by zowoq** [comment]
Updated to 36.0.

**Review by hedning — APPROVED** [review]
Did a quick sanity check that it runs in a nested gnome shell. LGTM at least :laughing:

**Comment by rawkode** [comment]
Lets get this merged! :smile:

**Comment by rawkode** [comment]
@hedning I think @NixOS/ofborg requested a review right after you reviewed this, could you approve this again and lets get it merged? :)

**Review by hedning — APPROVED** [review]

**Comment by hedning** [comment]
I'm happy to merge this myself, but thought I'd leave it to someone who's more impartial tbh.

**Review by rawkode — APPROVED** [review]

**Comment by rawkode** [comment]
@hedning I've approved it also, but no merge access.

**Comment by hedning** [comment]
Cheers :)

**Pull request merged**
